### PR TITLE
Killing Cloud Fix

### PR DESCRIPTION
**Pull request by MrKeiKun**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
Fixes the visual effect and flag of SO_CLOUD_KILL. Thanks @Asheraf 

**Issues addressed:** #2978 

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

File: src/map/skill.c [hunk 1]
```diff
@@ -11770,14 +11770,11 @@ static int skill_castend_pos2(struct block_list *src, int x, int y, uint16 skill
 			r = skill->get_splash(skill_id, skill_lv);
 			map->foreachinarea(skill->graffitiremover,src->m,x-r,y-r,x+r,y+r,BL_SKILL);
 			break;
-
 		case SO_WARMER:
-			flag|= 8;
-			/* Fall through */
 		case SO_CLOUD_KILL:
+			flag |= (skill_id == SO_WARMER) ? 8 : 4;
```
Comment (Asheraf) [inline src/map/skill.c:11775]: No sneaky changes.. if this is intended split it into it's own commit.

**Review by Asheraf — CHANGES_REQUESTED** [review]

File: src/map/skill.c (hunk 1 above)
Comment (MrKeiKun) [reply src/map/skill.c:11775]: yeah i will just split it...

**Review by MishimaHaruna — APPROVED** [review]
